pkgs/applications/office/ledger-autosync/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, lib, python3Packages, fetchurl }:
+
+python3Packages.buildPythonApplication rec {
+  version = "1.0.0";
+  src = fetchurl {
+    url = "https://github.com/egh/ledger-autosync/archive/v${version}.tar.gz";
```

Comment:
I wasn't able to find documentation on fetchFromGithub that told me what the sha256 hash is a hash of. Is it the git tree tarred and gzipped? It's not the commit sha hash..

Reply:
`fetchFromGitHub` uses `fetchzip` under the hood. A succinct description of what that does can be seen [here](https://github.com/NixOS/nixpkgs/blob/812a89f46e1a020768980be88071d4e0f025264f/pkgs/build-support/fetchzip/default.nix#L1).

> Is it the git tree tarred and gzipped? It's not the commit sha hash.

Simply put it's a hash of the content and not the archive, which is subject to change.

Reply:
> Can you pass a whole file tree to sha256sum? I'm a bit confused still.

The actual tooling you can you to get this hash would be
```
nix-prefetch-url --unpack https://github.com/egh/ledger-autosync/archive/v1.0.0.tar.gz
```

We do `--unpack` because it's hashed after the content is unpacked.

Though I tend to do the `trust on first use model` thingy, where you make `sha256` a probably wrong value
```
0000000000000000000000000000000000000000000000000000
```
then we replace it with the hash nix expected after calculating it. 
(that's in the output when you try to build it btw)